geometry_msgs/Pose p1
geometry_msgs/Pose p2
---
bool status